myapp/main.py

```diff
@@ -0,0 +1,46 @@
+import bokeh
+from bokeh.layouts import widgetbox
+from bokeh.models.widgets import *
+from bokeh.plotting import curdoc, show, figure
+from bokeh.embed import file_html, components
+import json
+import pps_regressor
+import pandas as pd
+
+with open('meanpps.json', 'r') as nbhdpps:
+        nbhd_dict = json.load(nbhdpps)
+meanpps = pd.DataFrame(nbhd_dict, index=None)
+hoods = list(meanpps.index)
+
+def update_data(event):
+    bldg_data = {
+        'Neighborhood': str(float(nbhd.value)),
+        'Building-Type-Custom': density.active,
+        'Year Built': int(yearbuilt.value_input),
+        'Building-Grade': segment.active,
+        'Square Footage': squarefootage.value,
+        'Renovation': reno_new.active,
+        'Quarter': quarter.active,
+        'Year': 2019}
+    if reno_new.active == 1:
+        bldg_data['Year Built'] = 2019
+    else:
+        bldg_data['Year Built'] = int(yearbuilt.value_input)
+    price_pred = pps_regressor.prediction(bldg_data)
+    p.text = '$' + str(price_pred[0] * bldg_data['Square Footage'])
+
+# Create some widgets
+nbhd = Select(title="Neighborhood:", options=list(hoods))
```

Comment:
I'm not totally sure, but I think you can put this in an `if __name__ == '__main__'` clause